README.md

```diff
@@ -14,22 +14,26 @@ Images are always based on the latest Ubuntu image available on
 ## How to run an image
 
 All images are available in
-[Docker Hub under @schneiderfelipe](https://hub.docker.com/u/schneiderfelipe),
+[GitHub Container Registries](https://github.com/orgs/geem-lab/packages?repo_name=docker-hpc),
 so the command below does everything for you:
 
-    $ docker run -it schneiderfelipe/siesta:4.0.2 bash
+    $ docker run -it ghcr.io/geem-lab/IMAGENAME:TAGNAME /bin/bash
```

Comment:
Can we have a concrete example here? (It can be ORCA or any other software.)

I think it is nice to have a one-liner in the README.md that can be copy-pasted and just works®.